### ISSUE TITLE
Cucumber Teamcity::Cucumber::Formatter

### ISSUE THREAD

**Issue by AllenAbolmaali** [cucumber/common#1874]
When I try to Run a Scenario inside a feature file the system does not run the scenario and gives the below Error: 
--format Teamcity::Cucumber::Formatter

Process finished with exit code 0

How ever when I click on the name.feature file and run - it runs perfectly .  only happens on 1 project and ather project work as expected - I am using same SDK path

**Comment by aurelien-reeves** [comment]
Hi,

We miss some info to be able to reproduce your issue

Could you please give more info? More trace? The whole command with the full error from the console?

**Comment by aslakhellesoy** [comment]
Who maintains `Teamcity::Cucumber::Formatter`? Is it JetBrains? Shouldn't this be reported to those maintainers?

**Comment by luke-hill** [comment]
Yep that's the inbuilt formatter. Each time there is a potential ruby / cucumber upgrade (or java upgrade e.t.c.), it could break.

**Comment by mpkorstanje** [comment]
I've taken to listing the incompatible plugins:

https://github.com/cucumber/cucumber-jvm/blob/254041f05f6529e7c29a826ac8a9a50b21494514/core/src/main/java/io/cucumber/core/options/PluginOption.java#L62-L83

And then complaining when they're used:

https://github.com/cucumber/cucumber-jvm/blob/254041f05f6529e7c29a826ac8a9a50b21494514/core/src/main/java/io/cucumber/core/options/PluginOption.java#L120-L130

Though for IDEA I do silently replace their implementation with my own

**Comment by aslakhellesoy** [comment]
That's a great way to deal with it @mpkorstanje!
